zadanie_8_9/zadanie_8_9.js

```diff
@@ -0,0 +1,24 @@
+var wiersze = prompt('Podaj liczbę wierszy: '),
+	i,
+	k,
+	j;
```

Comment:
dobra praktyka nakazuje, zeby powyzsze linie (ale patrz dalej) znalazly sie wewnatrz funkcji, poniewaz nalezy to do jej funkcjonalnosci.